scripts/anonymize.sql

```diff
@@ -186,8 +187,12 @@ UPDATE wiki_revisionip SET
 UPDATE wiki_revisionip SET
     referrer = CONCAT("https://example.com/", MD5(CONCAT(referrer, @common_hash_secret)))
     WHERE referrer != "";
+UPDATE wiki_revisionip SET data = null;
```

Comment:
Forgive my lack of context - just verifying the intent is to wipe the `data` field completely.